### ISSUE TITLE
Simplify styled-components support

### ISSUE THREAD

**Issue by nvh95** [nvh95/jest-preview#78]
**Is your feature request related to a problem? Please describe.**
Now, to support `styled-components`, we save to disk content of generated CSS. However, the generated CSS is in the `document.head` already. So we should use content from `document.head`, instead of re-generating CSS

**Describe the solution you'd like**
- Remove the code to generate and save `styled-components`'s CSS to disk.
- Use `document.head` instead.

**Describe how should jest-preview implements this feature**
Update `debug` function

**Comment by alexis-regnaud** [comment]
Does that mean the `styled-component`'s style will be handled out of the box ? 

Currently when I run `jest-preview` I am just seeing the HTM without any style displayed

**Comment by nvh95** [comment]
@alexis-regnaud yes. `styled-components` should work out of the box with Jest Preview. This issue is just an improvement to an existing implementation.

Can you provide a reproducible repository, I would love to help you debug. Also, can you create another issue?

**Comment by nvh95** [comment]
@alexis-regnaud This issue is going to be closed. Feel free to open a new issue if you still have an issue with `styled-components`.

If you are seeing HTML without any style, consider taking a look at [I can see HTML on my browser, but without any CSS
](https://www.jest-preview.com/docs/others/faq#i-can-see-html-on-my-browser-but-without-any-css) in FAQ.
